src/models/device.ts

```diff
@@ -368,7 +378,11 @@ const getDeviceModel = function (
 			{ primitive: true, promise: true },
 		);
 
-		const osUpdateHelper = await getOsUpdateHelper();
+		const osUpdateHelper =
```

Comment:
Please add an internal comment that gives a short description of v1 vs. v2 so the reader will have an idea of the purpose.